management/user_test.go

```diff
@@ -483,9 +570,26 @@ func givenAUser(t *testing.T) *User {
 	return user
 }
 
+func givenRefreshTokens(t *testing.T) *RefreshTokenList {
+	t.Helper()
+	user := &User{ID: auth0.String("UserID")}
+
+	tokens, err := api.User.GetRefreshTokens(context.Background(), user.GetID())
+	require.NoError(t, err)
+	return tokens
+}
+
 func cleanupUser(t *testing.T, userID string) {
 	t.Helper()
 
 	err := api.User.Delete(context.Background(), userID)
 	require.NoError(t, err)
 }
+
+func skipE2E(t *testing.T) {
```

Comment:
```suggestion
func skipTestIfRunningE2E(t *testing.T) {
```

Let's be a bit more descriptive as to what this does.